### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by nkverse**
## Desktop version layout

Changes and media queries to make the website displays well on desktop devices

### CODE REVIEW

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @nkverse , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost2

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️

- Kindly, I suggest you use the README file format provided by Microverse with the sections that apply to this project, it is important that you place your name and contact information in this presentation.
- kindly i suggest you to use flexbox in this section, there you can center elements in the container and it will fit any device size. (This will prevent the text from spilling out of the container)

check this section:
section.main {
...
  padding-top: 12rem;
}

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,20 +1,11 @@
...
 
 
 ## How to?
 
-**Important**: please remember to inspect the design in a mobile
-version size. i.e. `tested in 400px width`
-
 > `clone this repo`
 >
 > open `index.html`
```
Comment (TadesseDev) [inline README.md]: - [ ] Great on making some changes to the README file. It's one of the main requirements to have a standard readme file. Kindly, make sure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo.  That includes:-
- Project Name
- Built With
- Live Demo (if available)
- Getting Started
- Authors
-  Contributing
- Show your support
- Acknowledgments
- License
- > Note: you may leave some details empty if yet to be implemented. Like the `live demo` and `licence`

**Review by TadesseDev — CHANGES_REQUESTED** [review]
Hi @nkverse 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

bbcan-bbcan3


## Highlights
- Good job making some of the changes. :heavy_check_mark:
- Professional-looking readme file. :heavy_check_mark:
- Clear and descriptive Pull Request. :heavy_check_mark:
- Clean and tidy code. :heavy_check_mark:
- Good job following the design. :heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (nkverse) [reply README.md]: Done! Thank you @Tadesse-Alemayehu

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @nkverse 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - Very good UI design. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] The template 'Works' section cards are divided half of it contains the project's picture and the other half contains the project's description, these switch places with each project (the picture is on the left and the description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth). It would be great if your project was like that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**